analyzers/src/SonarAnalyzer.Common/Rules/SymbolicExecutionRunnerBase.cs

```diff
@@ -72,27 +72,28 @@ protected static bool IsEnabled(SonarSyntaxNodeReportingContext context, Diagnos
         }
     }
 
-    protected void Analyze<TNode>(SonarAnalysisContext analysisContext, SonarSyntaxNodeReportingContext context, Func<TNode, SyntaxNode> getBody) where TNode : SyntaxNode
+    protected void Analyze(SonarAnalysisContext analysisContext, SonarSyntaxNodeReportingContext context)
     {
-        if (getBody((TNode)context.Node) is { } body && context.SemanticModel.GetDeclaredSymbol(context.Node) is { } symbol)
+        if (context.SemanticModel.GetDeclaredSymbol(context.Node) is { } symbol)
         {
-            Analyze(analysisContext, context, body, symbol);
+            Analyze(analysisContext, context, symbol);
         }
     }
 
-    protected void Analyze(SonarAnalysisContext analysisContext, SonarSyntaxNodeReportingContext nodeContext, SyntaxNode body, ISymbol symbol)
+    protected void Analyze(SonarAnalysisContext analysisContext, SonarSyntaxNodeReportingContext nodeContext, ISymbol symbol)
     {
-        if (body is { ContainsDiagnostics: false })
+        var node = nodeContext.Node;
+        if (node is { ContainsDiagnostics: false })
         {
-            AnalyzeSonar(nodeContext, body, symbol);
+            AnalyzeSonar(nodeContext, node, symbol);
```

Comment:
This looks useless now
```suggestion
            AnalyzeSonar(nodeContext, symbol);
```